gammapy/datasets/utils.py

```diff
@@ -14,13 +16,43 @@ def apply_edisp(input_map, edisp):
     input_map : `~gammapy.maps.Map`
         The map to be convolved with the energy dispersion.
         It must have an axis named "energy_true".
-    edisp : `gammapy.irf.EDispKernel`
+    edisp : `~gammapy.irf.EDispKernel`
         Energy dispersion matrix.
 
     Returns
     -------
     map : `~gammapy.maps.Map`
         Map with energy dispersion applied.
+
+    Examples
+    --------
+    >>> from gammapy.irf.edisp import EDispKernel
+    >>> from gammapy.datasets.utils import apply_edisp
+    >>> from gammapy.maps import MapAxis, Map
+    >>> import numpy as np
+    >>>
+    >>> axis = MapAxis.from_energy_bounds("1 TeV", "10 TeV", nbin=6, name="energy_true")
+    >>> m = Map.create(
+    ...     skydir=(0.8, 0.8),
+    ...     width=(1, 1),
+    ...     binsz=0.02,
+    ...     axes=[axis],
+    ...     frame="galactic"
+    ... )
+    >>> m.data = np.arange(m.data.size, dtype=float).reshape(m.geom.data_shape)
```

Comment:
Why this line? This is super counter intuitive... and geeky!

Reply:
This is not actually needed, I accidentally left it there when I was testing other things. I have removed it!